### ISSUE TITLE
Problem on Upgrade native module if no logged on Addons

### ISSUE THREAD

**Issue by sebastienhouzet** [PrestaShop/PrestaShop#12623]
**Describe the bug**
If you want to upgrade native module with Admin or CLI, the source code is upgraded but not the version on PREFIX_module table.

**To Reproduce**
Install version 1.7.4.0
On CLI uninstall module ps_facetedsearch
Go on Admin > tabs module > Selection
search ps_facetedsearch
And Install ps_facetedsearch

Version on Admin is 2.1.2 and upgrade to 2.2.1 is propose.

Click on Upgrade, you source code is now 2.2.1 but on database module is 2.1.2.
And you have confirmation success upgrade message :/

If you tried with CLI

`php bin/console prestashop:module upgrade ps_facetedsearch`

You have this error 
ERROR     [console] Error thrown while running command "prestashop:module upgrade ps_facetedsearch". Message: "Error sent by Addons. You may need to be logged." ["exception" => Exception { …},"command" => "prestashop:module upgrade ps_facetedsearch","message" => "Error sent by Addons. You may need to be logged."] []

**Screenshots**
Source code is upgraded
![source](https://user-images.githubusercontent.com/2550639/53011966-f8035f80-3441-11e9-9a46-42c08bdda028.png)

But not database
<img width="303" alt="capture d ecran 2019-02-19 a 12 27 30" src="https://user-images.githubusercontent.com/2550639/53011878-c12d4980-3441-11e9-9a78-1aebc813684a.png">

On Admin, prestashop propose to upgrade
<img width="1255" alt="capture d ecran 2019-02-19 a 12 28 16" src="https://user-images.githubusercontent.com/2550639/53011932-e02bdb80-3441-11e9-8c46-09d20c113ec2.png">

**Additionnal information**
PrestaShop version: 1.7.4
PHP version: 7.1

**Comment by khouloudbelguith** [comment]
Hi @sebastienhouzet,

About the first issue, I did not manage to reproduce the issue with PS1.7.4.4.
I attached a video record.
https://drive.google.com/file/d/19j3tVGYZoFqOtDsn8McPh4VHIhN_PX4e/view
The version of the module is well updated in the Database.
About the second issue, after clearing cache manually by deleting all the folder into var/cache (prod & dev).
I tried to use this command
`php bin/console prestashop:module upgrade ps_facetedsearch` with PS1.7.5.1.

![image](https://user-images.githubusercontent.com/16067358/53019032-212ceb80-3454-11e9-8b99-89d78a31ba73.png)
And it is OK.

Thanks to check & feedback.
